File: Modul-6/Soal-5/PRAK605-2410817310012-FAISALTANJUNG.c
#include <stdio.h>
int main() {
    int n;
    scanf("%d", &n);

    int A[n][n], B[n][n], AXB[n][n];

    printf("matriks A\n");
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            scanf("%d", &A[i][j]);
        }
    }

    printf("matriks B\n");
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            scanf("%d", &B[i][j]);
        }
    }
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            AXB[i][j] = 0;
        }
    }
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            for (int k = 0; k < n; k++) {
                AXB[i][j] += A[i][k] * B[k][j];
            }
        }
    }
    printf("Matriks AXB:\n");
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            printf("%d ", AXB[i][j]);
        }
        printf("\n");
    }

    return 0;
}

File: Modul-6/Soal-5/PRAK605-2410817310012-FAISALTANJUNG.py
n = int(input())
print("Matriks A")

A = []
for i in range(n):
    A.append(list(map(int, input().split())))

print("Matriks B")
B = []
for i in range(n):
    B.append(list(map(int, input().split())))

AXB = [[0 for _ in range(n)] for _ in range(n)]

for i in range(n):
    for j in range(n):
        for k in range(n):
            AXB[i][j] += A[i][k] * B[k][j]

print("Matriks AXB")
for row in AXB:
    print(" ".join(map(str, row)))
